arenadata/readme.md

```diff
@@ -22,16 +22,13 @@ We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_g
 
 ```bash
  docker run --name gpdb7_opt_on --rm -it -e TEST_OS=centos \
-  -e MAKE_TEST_COMMAND=-k PGOPTIONS='-c optimizer=on' installcheck-world \
-  --sysctl kernel.sem=500 1024000 200 4096 gpdb7_regress:latest \
-  scl enable devtoolset-7 llvm-toolset-7 \
+  -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
+  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_regress:latest \
   /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash
 ```
 
 * we need to modify `MAKE_TEST_COMMAND` environment variable to run different suite. e.g. we may run test againt Postgres optimizer or ORCA with altering `PGOPTIONS` environment variable;
-* we need to run container as `--privileged` to run debugger inside it
```

Comment:
Why do you consider this statement as redundant? Are you able to run gdb inside an unprivileged container?

Reply:
yes

Reply:
Really?

```
[root@ac4d63bf750d gpadmin]# gdb -p 13079 --batch --ex 'bt 10'
ptrace: Operation not permitted.
No stack.
```